scribejava-apis/src/main/java/com/github/scribejava/apis/DoximityApi20.java

```diff
@@ -0,0 +1,64 @@
+package com.github.scribejava.apis;
+
+import com.github.scribejava.apis.service.Doximity20ServiceImpl;
+import com.github.scribejava.core.builder.api.DefaultApi20;
+import com.github.scribejava.core.model.OAuthConfig;
+import com.github.scribejava.core.model.OAuthConstants;
+import com.github.scribejava.core.model.ParameterList;
+import com.github.scribejava.core.oauth.OAuth20Service;
+
+import java.util.Map;
+
+public class DoximityApi20 extends DefaultApi20 {
+
+    protected DoximityApi20(){
+    }
+
+    private static class InstanceHolder {
+        private static final DoximityApi20 INSTANCE = new DoximityApi20();
+    }
+
+    public static final DoximityApi20 instance(){
+        return InstanceHolder.INSTANCE;
+    }
+
+    @Override
+    public String getAccessTokenEndpoint() {
+        return "https://auth.doximity.com/oauth/token?grant_type=authorization_code";
```

Comment:
?grant_type=authorization_code looks redundant
It's standard OAuth2 param, it will be added here https://github.com/scribejava/scribejava/blob/master/scribejava-core/src/main/java/com/github/scribejava/core/oauth/OAuth20Service.java#L85